spec/slo.cue

```diff
@@ -0,0 +1,6 @@
+package v1alpha1
+
+apiVersion: "openslo/v1alpha1" & #apiVersion
+kind:       "SLO" & #kind
```

Comment:
For `Service` the spec is rather brief on what goes into it (besides `spec.Description`) so I did not include `Service` type yet.